q2_cutadapt/tests/test_demux.py

```diff
@@ -457,7 +457,7 @@ def test_build_demux_command(self):
                          obs[11])
 
     def test_rename_files_single(self):
-        for fn in ['sample_a.fastq.gz', 'sample_b.fastq.gz']:
+        for fn in ['sample_a.1.fastq.gz', 'sample_b.1.fastq.gz']:
```

Comment:
The assertions in this test were not actually running because the file names were incorrect (assuming this was unintentional?).

Reply:
Yep, good catch, looks like it was a typo.

One way to fix this:

```python
seqs = self.per_sample_dir_fmt.sequences.iter_views(FastqGzFormat)
counter = 0
for fn, (sample_id, barcode) in zip(seqs,
                                    self.barcode_series.iteritems()):
            self.assertTrue(sample_id in str(fn))
            self.assertTrue(barcode in str(fn))
            counter = counter + 1
self.assertEqual(counter, 2)
```